src/pint/fitter.py

```diff
@@ -160,13 +160,27 @@ def get_summary(self, nodmx=False):
         import uncertainties.umath as um
         from uncertainties import ufloat
 
+        # Check if Wideband or not
+        if self.resids.__class__.__name__ == "WidebandTOAResiduals":
```

Comment:
Because the Fitter constructs the Residuals, only the WidebandTOAFitter will ever construct residuals that need this branch. A more natural approach would be to use the inheritance tree and the fact that methods in WidebandTOAFitter override the versions in Fitter. That would require breaking this method into smaller methods - one for each hunk where the wideband fitter differs from the Fitter. I'm not sure that this solution is a very good one.

Reply:
Ok; then we'll go with this simpler fix for now.